config/default.js

```diff
@@ -242,6 +242,19 @@ module.exports = {
   },
   rtlLangs: ['ar', 'dbr', 'fa', 'he', 'ur'],
   defaultLang: 'en-US',
+  // Mapping for missing moment locales that we can still map to existing ones.
+  // Moment locales are lowercase and do not use an underscore.
+  // See: https://github.com/mozilla/addons-frontend/issues/1515
+  momentLangMap: {
+    'bn-bd': 'bn',
+    'fy-nl': 'fy',
+    'nb-no': 'nb',
+    'nn-no': 'nn',
+    'pt-pt': 'pt',
+    'sv-se': 'sv',
+    dbl: 'x-pseudo',
+    dbr: 'x-pseudo',
```

Comment:
given that these are being removed in #6624, do we need to include dbl and dbr here?

Reply:
It depends on which PR lands first 😋